packages/devtools_app/lib/src/app.dart

```diff
@@ -459,6 +461,10 @@ List<DevToolsScreen> get defaultScreens => <DevToolsScreen>[
         const LoggingScreen(),
         createController: () => LoggingController(),
       ),
+      DevToolsScreen<CodeSizeController>(
```

Comment:
put a top level boolean in code_size_screen.dart `codeSizeScreenEnabled` that defaults to false. Then only add the screen here if that boolean is true. This will ensure the code size screen doesn't leak into any builds before it is ready

Reply:
yep exactly. until the page is ready to be turned on by default